### ISSUE TITLE
Plant Person Mech Light

### ISSUE THREAD

**Issue by DavidEdwards** [yogstation13/yogstation-classic#725]
Currently a Plant Person will lose life in a Mech. This is an intended side affect of not having enough light when inside a Mech.

We should have a module available for Mechs that enhance lifesupport to include a bright light for Plant People.

As suggested on Discord:

```
[11:22 AM] ToGWtF: Upgraded lift support module
[11:22 AM] ToGWtF: Gives an internal light for plants, and a larger internal oxygen syste
[11:22 AM] ToGWtF: life support*
```

I think this is a neat idea. Requiring a Plant Person to sacrifice a slot to be able to remain in their Mech with no issues.

@yogstation13/initiate-coders this should be a nice little project for someone.

**Comment by AsV9** [comment]
I swear I fixed this in an earlier PR. #661 I think

**Comment by DavidEdwards** [comment]
Would you mind double checking in a Mech?

```
[10:21 AM] Hydre: So are we aware that plant people die in exosuits?
[10:24 AM] X-TheDark: You mean mechs?
[10:25 AM] X-TheDark: I thought that was fixed.
[10:25 AM] Hydre: Nope
[10:26 AM] Hydre: Plant peep had to carry a hand flashlight inside the mech to not die, and even then had to occasionally get out of the mech because he started dying anyways
```

**Comment by AsV9** [comment]
Probably fucked the if check then.

**Comment by LesVegetables** [comment]
In regards to the earlier bug fix (#661) it also seems plants still take damage on the syndie shuttle as of a round ~12 hours ago one went into crit and got saved by me as a syndieborg. It may have something to do with wearing your hardsuit in combat mode though, because I remember it working in a round where a syndie cabbage on the shuttle wasn't wearing one and was fine. I'll try to test this theory later on today.

Would it be possible to just spawn 5 flashlights as a duct tape fix incase admins are not on? The desk lamp doesn't seem to provide light at the moment except when it is in your hand, regardless of its state.
